### ISSUE TITLE
User arborted

### ISSUE THREAD

**Issue by papermate** [dyne/Tomb#202]
Hi guys,
I just installed tomb (v. 2.0.1) from tar on my headless raspberry pi 

The installation was flawless, all the dependencies are met.
Not sure is me missing something or an actual bug.

When I dig all is fine:
```
tombom@pi:~/tmp$ sudo tomb dig -s 10 test.tomb
tomb  .  Commanded to dig tomb test.tomb
tomb (*) Creating a new tomb in ./test.tomb
tomb  .  Generating test.tomb of 10MiB
10+0 records in
10+0 records out
10485760 bytes (10 MB) copied, 7.31928 s, 1.4 MB/s
-rw------- 1 root root 10M May 27 09:56 ./test.tomb
tomb (*) Done digging test
tomb  .  Your tomb is not yet ready, you need to forge a key and lock it:
tomb  .  tomb forge ./test.tomb.key
tomb  .  tomb lock ./test.tomb -k ./test.tomb.key
```
But when I forge the key something goes wrong, after I've been prompt for a password:
```
tombom@pi:~/tmp$ sudo tomb forge ./test.tomb.key
tomb  .  Commanded to forge key ./test.tomb.key
tomb  .  Commanded to forge key ./test.tomb.key with cipher algorithm AES256
tomb  .  This operation takes time, keep using this computer on other tasks,
tomb  .  once done you will be asked to choose a password for your tomb.
tomb  .  To make it faster you can move the mouse around.
tomb  .  If you are on a server, you can use an Entropy Generation Daemon.
256+0 records in
56+0 records out
256 bytes (256 B) copied, 402.318 s, 0.0 kB/s
tomb (*) Choose the  password of your key: ./test.tomb.key
tomb  .  (You can also change it later using 'tomb passwd'.)
pinentry-curses: no LC_CTYPE known - assuming UTF-8
tomb [W] Pinentry error: syntax
tomb [E] User aborted.
```

**Comment by jaromil** [comment]
Hi there,

searching the error, this thread helps to understand https://forums.gentoo.org/viewtopic-t-699598-start-0.html

```
# GPG always wants to know what TTY it's running on. 
export GPG_TTY=`tty`
```

**Comment by jaromil** [comment]
wondering if we should add this env setting to the pinentry wrapper in Tomb?

**Comment by papermate** [comment]
Hi jaromil, thanks for your reply. Unfortunately your hint did not work, but I tried a lot of things and probably the problem is in gnupg which does not recognize/use pinentry for some reason I don't understand, not in Tomb.

**Comment by papermate** [comment]
Just in case someone has the same issue, I found a way to have tomb work. I tried a lot of things before, seriously.
Just give a 
''' 
eval $(gpg-agent --daemon --pinentry-program=/usr/bin/pinentry-curses)
'''
before the forge

I may also add that, being on a headless server, installing this less-than-200 Kb package got me from couple of minutes to 3 seconds for the forge. It keeps the random pool full.
'''
sudo apt-get install get haveged
'''

Thanks for your support and for the really nice software!
